io/net/virt-net/lpm_basic.py.data/lpm_basic.yaml

```diff
@@ -0,0 +1,3 @@
+hmc_pwd: abc123
+hmc_username: hscroot
+remote_server: lpm1br1fp1
```

Comment:
Please remove the  static values from yaml file.

Reply:
Removed actual system name and added dummy name.